helpers/cleanup.sh

```diff
@@ -142,7 +142,7 @@ gcloud projects remove-iam-policy-binding "${PROJECT_ID}" \
 
 gcloud projects remove-iam-policy-binding "${PROJECT_ID}" \
     --member="serviceAccount:${SERVICE_ACCOUNT_EMAIL}" \
-    --role="roles/compute.networkViewer" \
+    --role="roles/compute.networkAdmin" \
```

Comment:
Should this always be required? If someone is deploying Forseti with public IPs, then this is not required.

Reply:
It's worth discussing.  If our going in position is that you should always deploy Forseti with private IP's then I say, yes, this is always required.